src/fs/create.js

```diff
@@ -1,8 +1,9 @@
+import path from 'node:path';
 import fs from 'node:fs/promises';
 
 const create = async () => {
 
-    const fileName = 'files/fresh.txt';
+    const fileName = path.join('files', 'fresh.txt');
```

Comment:
this path won't work anywhere but locally on your pc, otherwise you would receive 'ENOENT' error and it won't work. I suggest you can change it to the `path.join(path.dirname(new URL(import.meta.url).pathname),'files/fresh.txt');`
on other files too

Reply:
You still have time to change it, so i'll review your task later

Reply:
Could you please test it as highlighted in bold (I'm afraid to commit after the deadline)

> 5. Start to check subtasks according to the assignment description.
>    For check simplification students expected to have npm-scripts in `package.json` how to run every subtask.
>    If not, follow below instruction:
> 
> All tasks are located inside `src` folder. Open terminal in **src** folder of downloaded project.
> For example 1st subtask **create.js** located in _fs_ folder:
> 
> - Run `node fs/create.js` in your terminal opened in _src_ folder.
>   or **go into fs/ subfolder of the task and run `node create.js`**.

Reply:
Yep, i did it the same way it was written in cross-check tutorial file, however it didn't work. Without changing files final score score would be 64/

Reply:
I'm sorry I didn't understand why it doesn't work. Could you explain, why? I think it should work and it works

Reply:
It isn't working properly because you are using relative path instead of absolute. Relative path specifies the location of a file or directory in relation to the current working directory, while absolute path provides the full path from the root directory.

Paths may differ across devices due to varying file system structures. An ENOENT error occurs when a specified path is not found.This can happen if the file/directory structure is different on another device or if the path is incorrectly specified.

Reply:
We don't have requirement to use absolute paths, and in this project we have strict predefined file structure, so I don't agree that this is an error
You can run script create.js from fs subfolder and in this case it works. Cross-check tutorial let as check script from fs subfolder.